srcpkgs/mimalloc/patches/secure-basename.patch

```diff
@@ -0,0 +1,14 @@
+# Set basename to always be 'mimalloc' for compatibility
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 2550f0d..9dff9e9 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -292,7 +292,7 @@ endif()
+ 
+ set(mi_basename "mimalloc")
+ if(MI_SECURE)
+-  set(mi_basename "${mi_basename}-secure")
++  set(mi_basename "${mi_basename}")
+ endif()
```

Comment:
```suggestion
-if(MI_SECURE)
-  set(mi_basename "${mi_basename}-secure")
-endif()
```
This conditional doesn't make any sense. Please remove it completely.

Reply:
Done.